### DISCUSSION TITLE
Performance drop when mod select menu is opened

### DISCUSSION THREAD

**Discussion by denizberkin** [ppy/osu#23467] [Q&A] [answered]
# I get a major performance drop when opening the mod selection menu. The drop always happens but it gets huge when I don't open it for a while.

https://github.com/ppy/osu/assets/63101810/2b06be72-21ec-4a30-9d29-4754849f5432

I am not sure if this is on my end or not but kind of irritating to have.

The game has Direct3D 11 selected with no limit on the frame rate. I also play on 1920x1080 resolution.

## My hardware and software:
AMD Ryzen 5 3600 6-core CPU
Nvidia RTX 3060 GPU  driver is up to date version: 531.79
Windows 11 Pro version 22H2, 22624.1616

**Comment by peppy** [comment, marked as answer]
This is just due to a lot going on at once. We'll continue to optimise but there's nothing specific to be done until that happens.
